app/models/supplejack_api/user_set.rb

```diff
@@ -39,11 +40,10 @@ class UserSet
 
     validates :name, presence: true
 
-    before_save :strip_html_tags!
```

Comment:
Was this not used anymore?

Reply:
I have grouped them together :D

Reply:
Ah true, I was guessing so but I don't see the update in the code, is it not in the same file?